### PR TITLE
Update card components

### LINKED ISSUE #2581: Updated Card Components
### Description

Update card components per recent usability feedback in an effort to make Detection, Alert, and Log source cards more informational without having to click into the details page.

### Designs

Designs are [here](https://app.abstract.com/projects/558a8de6-7134-4c8b-91c5-62074bb1279b/branches/df9d59c8-eeb1-452d-a5f5-3712b64010be/commits/0c52e7c01e4403134ed1a4265348369827334643/files/29396687-8ca0-4c55-a4c7-2399489f10a9/layers/9DF07065-5715-406A-9D19-E731CD57539D)

### Acceptance Criteria

- Update coloring per new designs
- Add in # of events to Rules Error/Rules Match/Scheduled Rules Match/Scheduled Rules Error alert cards
- Add in Resource name to Policy Fail alert card
- Display error message when alert delivery is not successful
- Display when data was last received on Log Source cards

### PR DESCRIPTION
**Pull request by kouknick**
## Description

Update card components as per recent usability feedback. 

Closes #2581

#### Changes

- Update coloring per new designs
- Add in # of events to Rules Error/Rules Match/Scheduled Rules Match/Scheduled Rules Error alert cards
- Add in Resource name to Policy Fail alert card
- Display when data was last received on Log Source cards

### Screenshots:

https://user-images.githubusercontent.com/15084305/107652585-6b36b980-6c89-11eb-8d9d-41117ec932bb.mov

### Checklist:

- [x] Make sure you give an high-level overview of the problem & the solution
- [x] Make sure the related issue is referenced
- [x] Make sure screenshots are added if the UI is changed in any way
- [x] Make sure all of the significant logic is covered by tests
- [x] Make sure your changes are accessible (addition of aria-attributes & correct HTML semantics)
- [x] Make sure you reference the related design document if it exists
- [x] Make sure telemetry is added (analytics & errors)

### CODE REVIEW

**Comment by kouknick** [comment]
Unfortunately, the designs for the `Destinations` and `Log Analysis Sources` are not yet finished, they will be updated in the next sprint.

**Review by vorillaz — APPROVED** [review]
AMAZING work @kouknick

File: web/src/components/GenericItemCard/GenericItemCard.tsx [hunk 1]
```diff
@@ -76,12 +80,43 @@ const GenericItemCardHeader: React.FC = ({ children }) => {
...
+            {label}
+          </Text>
+        )}
+      </Box>
+      {withDivider && (
+        <Text px={2} fontSize="small" as="span" color="navyblue-100">
+          &bull;
+        </Text>
+      )}
+    </>
```
Comment (3nvi) [inline web/src/components/GenericItemCard/GenericItemCard.tsx:116]: IMO This shouldn't be part of the component. It has no reason to. You can create a separrate component named `GenericItemCardBulletDivider` and use it when you need to

File: web/src/components/GenericItemCard/GenericItemCard.tsx [hunk 2]
```diff
@@ -76,12 +80,43 @@ const GenericItemCardHeader: React.FC = ({ children }) => {
...
 const GenericItemCardHeading: React.FC<TextProps> = ({ children, ...rest }) => {
   return (
-    <Text as="h4" fontWeight="medium" mr="auto" maxWidth="70%" wordBreak="break-word" {...rest}>
+    <Text as="h4" fontWeight="medium" mr="auto" maxWidth="60%" wordBreak="break-word" {...rest}>
       {children}
     </Text>
   );
 };
 
+const GenericItemCardHeadingValue: React.FC<GenericItemCardHeadingValueProps> = ({
```
Comment (3nvi) [inline web/src/components/GenericItemCard/GenericItemCard.tsx:89]: naming is a bit misleading. I feel that this component's name should be `GenericItemMetaValue` or smth

**Pull request merged**